tests/network-tests/src/flows/membership/invitingMembers.ts

```
@@ -18,6 +22,37 @@ export default async function invitingMembers({ api, query, env }: FlowProps): P
   await new FixtureRunner(buyMembershipHappyCaseFixture).run()
   const [inviterMemberId] = buyMembershipHappyCaseFixture.getCreatedMembers()
 
+  // Membership WG balance required
```

Since this flow could be used in other scenarios where perhaps a council was not elected, maybe it should fail early.